app/decorators/service_provider_session_decorator.rb

```diff
@@ -117,6 +117,10 @@ def url_options
     end
   end
 
+  def irs_attempts_api_enabled
+    sp.irs_attempts_api_enabled || false
+  end
```

Comment:
why the `|| false`? Is this to avoid `nil`? I think in general nil is fine.

Also two ideas:
1. Either make this a delegated method
2. Add a `?` to make it a predicate method:

```suggestion
  def irs_attempts_api_enabled?
    sp.irs_attempts_api_enabled
  end
```

Reply:
Done - thanks!